packages/suite/src/support/extraDependencies.ts

```diff
@@ -124,6 +128,18 @@ export const extraDependencies: ExtraDependencies = {
                 state.historic = historicRates;
             }
         },
+        storageLoadTokenManagement: (
+            state: TokenDefinitionsState,
+            { payload }: StorageLoadAction,
+        ) => {
+            if (payload.tokenManagement) {
+                const tokenDefinitions = buildTokenDefinitionsFromStorage(payload.tokenManagement);
+                Object.keys(tokenDefinitions).forEach(networkSymbol => {
+                    state[networkSymbol as NetworkSymbol] =
+                        tokenDefinitions[networkSymbol as NetworkSymbol];
+                });
+            }
+        },
```

Comment:
The types of `tokenDefinitions` and `state` are correct. This is because `Object.keys` will make the `forEach` variable a string so the best I think I can do here is to do like:

```
Object.keys(tokenDefinitions).forEach(networkSymbol => {
    const symbol = networkSymbol as NetworkSymbol;
    state[symbol] = tokenDefinitions[symbol];
});
```

Reply:
there is `TODO: fix types` in `buildTokenDefinitionsFromStorage` so thats why I thought that it can be fixed

Reply:
a7326497453e1d72bd168f7fbfbc818ad3072841